### ISSUE TITLE
Building for aarch64/gles/sdl2 after commit 6f2abdac75e115a937fa2aa60d714db8166fa900 fails with missing (previously not needed) glew (and other errors)

### ISSUE THREAD

**Issue by shantigilbert** [Rubberduckycooly/Sonic-1-2-2013-Decompilation#291]
Using commit bff61dff624ee986ace719583b00baab22a40d52 I can cross-compile fine on Linux for aarch64/GLES2/SDL2 and it seems it plays and works fine. 

But from commit 6f2abdac75e115a937fa2aa60d714db8166fa900 and after, building complains that it needs glew, I tried to follow the changes to see if I could figure something out, but honestly (and no disrespect) I gave up because the commit messages are extremely hard to follow (some of them are funny, but not useful for debugging) so here I am opening an issue asking for an idea on how I could fix this, or is glew/x11/OpenGL going to be needed now? if its a platform issue (as in `RETRO_PLATFORM`), how could I force it to use aarch64/gles/sdl2? 

This is the error (it repeats a few times): 

```
In file included from RSDKv4/Audio.cpp:1:
RSDKv4/RetroEngine.hpp:173:10: fatal error: GL/glew.h: No such file or directory
  173 | #include <GL/glew.h>
      |          ^~~~~~~~~~~
```

Since this is a minimal distro (EmuELEC) that is run on low/med power devices (Odroid N2, Odroid Go advance, Gameforce Chi, Amlogic s905x3, etc, etc) we don't use glew/opengl/x11 

Thanks in advance

**Comment by MegAmi24** [comment]
The decomp uses GLEW to render the hardware/mobile main menu. Without it, you'd just get a black screen until the game engine starts.

**Comment by shantigilbert** [comment]
> The decomp uses GLEW to render the hardware/mobile main menu. Without it, you'd just get a black screen until the game engine starts.

Oh, that makes sense. Thanks!

So I assume this new main menu is forced and not possible to skip as it was before?

**Comment by MegAmi24** [comment]
There's an option to skip it, however it can only be toggled through the decomp's settings.ini file, not as a compiling option.

**Comment by shantigilbert** [comment]
> There's an option to skip it, however it can only be toggled through the decomp's settings.ini file, not as a compiling option.

Bummer... :( and are there any plans to make this optional while compiling?  I ask because this works fine otherwise (and in my case we really do not need that menu) and unfortunately I do not have the knowledge to remove the menu or add an "ignore" option.

**Comment by MegAmi24** [comment]
The decomps are finished and are no longer being worked on aside from minor bug fixes.

**Comment by shantigilbert** [comment]
> The decomps are finished and are no longer being worked on aside from minor bug fixes.

Thank you, I will keep using the old commit and hope for the best :P 

I will close this issue now!
